File: 35.js
function countValidSplits(s) {
    let ans = 0;
    for (let i = 1; i < s.length; i++) {
        let left = s.slice(0, i);
        let right = s.slice(i);
        if (left <= right) ans++;
    }
    return ans;
}

// Driver code
const fs = require("fs");
const s = fs.readFileSync(0, "utf8").trim();
console.log(countValidSplits(s));

File: 35.py
def count_valid_splits(s: str) -> int:
    n = len(s)
    ans = 0
    for i in range(1, n):
        left = s[:i]
        right = s[i:]
        if left <= right:
            ans += 1
    return ans

# Driver code
if __name__ == "__main__":
    s = input().strip()
    print(count_valid_splits(s))
